src/changelog.rs

```diff
@@ -9,6 +9,13 @@ use crate::*;
 /// ## Added
 ///  - Added changelog
 ///  - Added [`SessionBuilder::compression`]
+///  - Feature `scp`.
+///  - New mod [`scp`], contains the old SFTP implementation.
+///  - Added [`Session::scp`].
+///
+/// ## Removed
+///  - `Sftp`, `Mode` and `RemoteFile`.
```

Comment:
These weren't removed as far as I can tell?

Reply:
After mod `sftp` are renamed into `scp`, I removed these exports so now user have to type `scp::Sftp`.

Calling them removed is a little bit confusing, I will fix that.

Reply:
Done.